providers/base/bin/network.py

```diff
@@ -210,9 +210,13 @@ def extract_core_list(self, line):
         """Extract a list of CPU cores from a line of the form:
         NUMA node# CPU(s):    a-b[,c-d[,...]]"""
         colon = line.find(":")
-        cpu_list = line[colon + 1 :]
+        cpu_list = line[colon + 1:]
```

Comment:
`black` is complaining about this line, could you change it back?

```suggestion
        cpu_list = line[colon + 1 :]
```